pom.xml

```diff
@@ -152,11 +152,11 @@
             <plugin>
                 <groupId>com.mycila</groupId>
                 <artifactId>license-maven-plugin</artifactId>
-                <configuration combine.self="override">
-                    <excludes>
-                        <exclude>data/</exclude>
-                    </excludes>
-                </configuration>
+                <!--<configuration combine.self="override">-->
```

Comment:
Paul and I wrote this before, but it doesn't work, so I thought I removed it... Apparently I didn't